### PR TITLE
create getLEDCurrent() method

### PR DESCRIPTION
**Pull request by jrw429**
fixes issue 25, create getLEDCurrent()

### CODE REVIEW

File: Adafruit_AS7341.cpp [hunk 1]
```diff
@@ -502,6 +502,31 @@ bool Adafruit_AS7341::setLEDCurrent(uint16_t led_current_ma) {
...
+
+  Adafruit_BusIO_Register led_reg =
+      Adafruit_BusIO_Register(i2c_dev, AS7341_LED);
+
+  Adafruit_BusIO_RegisterBits led_current_bits =
+      Adafruit_BusIO_RegisterBits(&led_reg, 7, 0);
+
+  led_raw = led_current_bits.read();
+
+  led_current_ma = (uint16_t)(led_raw * 2) + 4;
```
Comment (caternuson) [inline Adafruit_AS7341.cpp:525]: Could just return right hand side here instead of creating `led_current_ma` and returning that.

**Review by caternuson — CHANGES_REQUESTED** [review]

**Comment by caternuson** [comment]
Looks good. Added a review comment.

Were you able to test this? Good test would be to set LED current, then read back and make sure it reports back as expected.

**Comment by jrw429** [comment]
Yes, tested and working.

File: Adafruit_AS7341.cpp (hunk 1 above)
Comment (jrw429) [reply Adafruit_AS7341.cpp:525]: Agreed. I tend to create "unnecessary" variables for the sake of being clear and documenting what the code does.

**Review by caternuson — APPROVED** [review]

**Pull request merged**